codec-http/src/test/java/io/netty/handler/codec/http/multipart/HttpPostRequestDecoderTest.java

```diff
@@ -768,7 +768,7 @@ public void testMultipartRequest() throws Exception {
     @Test
     public void testNotLeak() {
         final FullHttpRequest request = new DefaultFullHttpRequest(HttpVersion.HTTP_1_1, HttpMethod.POST, "/",
-                Unpooled.copiedBuffer("a=1&&b=2", CharsetUtil.US_ASCII));
+                Unpooled.copiedBuffer("a=1&=2&b=3", CharsetUtil.US_ASCII));
```

Comment:
Same here. 
The PR makes extraneous ampersands ignored. However if a parameter is defined with a key with no name it's still invalid; this test would fail because the new functionality means the previous test value is now no longer 'incorrect'. The updated test is still ensuring that there are no leaks when an invalid parameter is passed in the query params.